### DISCUSSION TITLE
changing bootnodes (in genesis file)

### DISCUSSION THREAD

**Discussion by dorijan** [0xPolygon/polygon-edge#1071] [General] [answered]
Hi to all,
is it possible to change bootnodes (or add more) to genesis file without restarting whole network?
I mean, I can restart all the nodes, but I would like to have all the transactions up.
when I tried to change them in genesis file, and restart all the nodes, nodes would not recoonect and got
`polygon.network.discovery: Unable to execute bootnode peer discovery`

Also, I see that --join no longer works, when I try this

```
polygon-edge server --join /ip4/192.168.xx.xx/tcp/10001/p2p/16Uiu2HAmFxxxxx
Error: unknown flag: --join

```

Thank you,
Dorijan

**Comment by dorijan** [comment, marked as answer]
to answer myself, I manged to change genesis file, but also had to put 
`--libp2p 0.0.0.0:10001 --nat x.x.x.x `
maybe this will help someone

**Comment by ivanbozic21** [comment]
Hi @dorijan 

Apology for the late response.

The --join flag is no longer in use, and our documentation was accordingly updated now.

Please know once you added a new bootnode entry in your genesis file, there is no need to restart your entire network and you can just restart a validator that is supposed to become a new bootnode.
